Plugins/Events/NWScript/nwnx_events.nss

```diff
@@ -1459,6 +1509,22 @@ const int NWNX_EVENTS_TIMING_BAR_LOCK          = 8;
 const int NWNX_EVENTS_TIMING_BAR_CUSTOM        = 10;
 */
 
+/*
+const int NWNX_EVENTS_DM_SET_STAT_TYPE_STRENGTH         = 5;
+const int NWNX_EVENTS_DM_SET_STAT_TYPE_DEXTERITY        = 6;
+const int NWNX_EVENTS_DM_SET_STAT_TYPE_CONSTITUTION     = 7;
+const int NWNX_EVENTS_DM_SET_STAT_TYPE_INTELLIGENCE     = 8;
+const int NWNX_EVENTS_DM_SET_STAT_TYPE_WISDOM           = 9;
+const int NWNX_EVENTS_DM_SET_STAT_TYPE_CHARISMA         = 10;
```

Comment:
If you subtract 5 from the stat value and push that, you can just use the basegame `ABILITY_*` constants

Reply:
Can do. Wasn't sure if you'd want the values to be left as is given by nwserver for the scripter to convert in nwscript themselves or not, -5 it is!